modules/claims_api/spec/requests/v2/veterans/rswag_claims_request_spec.rb

```diff
@@ -71,15 +96,20 @@
         end
       end
 
-      describe 'Getting a 404 response' do
-        let(:veteranId) { 'not-the-same-id-as-tamara' } # rubocop:disable RSpec/VariableName
-
-        response '404', 'Resource Not Found' do
+      describe 'Getting a 403 response' do
+        response '403', 'Resource Not Found' do
```

Comment:
should say "Forbidden" instead of "Resource Not Found"

Reply:
[D'oh!](https://github.com/department-of-veterans-affairs/vets-api/pull/7395/commits/f3dbe6bcd7a8a8e6125ec16a33803b74c508e295)
🤦